dll/Models/PostsPaged.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace dal.Models
+{
+    public class PostsPaged
```

Comment:
You can use this model not only for posts, but for everything paged if you will add a generic param here and use it as generic for underlying IList